.gitignore

```diff
@@ -8,3 +8,4 @@
 /gpii/node_modules/gpii-oauth2/gpii-oauth2-utilities/node_modules/
 
 npm-debug.log
+*.bak
```

Comment:
Is this something you put in for your own development purposes, or something you suggest we add as a general ignore clause?